### ISSUE TITLE
[Feature Req.] Add option to output real-space 21-cm boxes instead of redshift-space

### ISSUE THREAD

**Issue by JulianBMunoz** [21cmfast/21cmFAST#165]
For some applications users might want real-space 21-cm boxes, as opposed to redshift-space. Right now RSD (redshift-space distortions) are always applied, but it'd be a good enhancement to give the option to not do so.

**Comment by BradGreig** [comment]
Hey @JulianBMunoz, this feature already exists. Though, perhaps it isn't very well documented. There is a variable in the global params struct (`global_params.T_USE_VELOCITIES`) which by default is set to be 1 (i.e. True). Setting it to zero will turn off the addition of any velocities (i.e. will ignore velocity gradients and/or sub-cell RSDs).

I can agree its a bit messy this way, however, its complicated to have multiple velocity flags (i.e. toggling on/off velocities, while also being able to toggle on/off sub-cell RSDs). One could allow all combinations, which then makes it important which integer you pass to make sure you get the correct combination. Arguably, it is rare for velocities not to be required, thus having it in the current format.

**Comment by JulianBMunoz** [comment]
That does work, thank you! I did not know it was an option. 

Perhaps we could just document it a bit more, for instance the description of this flag in py21cmfast.inputs just reads "_Whether to use velocity corrections in 21-cm fields_", we could add a line saying "Setting it to False will not include redshift-space distortions (RSD), and output real-space coeval boxes.", or similar. Alternatively (or in addition), we could mention it in passing during the coeval_cubes tutorial. What do you think?

**Comment by BradGreig** [comment]
Yep, I definitely agree it could be documented better. I'll have a think about best ways to do this, though your suggests would probably suffice.
